cluster-operator/src/test/java/io/strimzi/operator/cluster/TestUtils.java

```diff
@@ -54,4 +56,30 @@ public static Map<String, String> containerEnvVars(Container container) {
                         // On duplicates, last-in wins
                         (u, v) -> v));
     }
+
+    /**
+     * If present, adds the HTTP/HTTPS proxy environment variables to the list.
```

Comment:
Maybe you can add a bit more here to better explain why and when it is needed => when your environment has these environment variables already set because running behind proxy etc.

Reply:
Done. Thanks.